### ISSUE TITLE
Feature request: Send message about reason for rejection

### ISSUE THREAD

**Issue by tobiasgerstenberg** [NYUCCL/psiTurk#176]
It would be great if the interface allowed me to send a message to an individual worker (or to several workers) with a reason for rejecting their work. For example, I sometimes reject workers who completed the task unreasonably fast (< 1 minutes when the median is > 10 minutes).

**Comment by ebigelow** [comment]
I agree, this would be very useful -- workers don't like being rejected without being given a reason.

**Comment by alexanderrich** [comment]
I just pushed some small changes that would let users input a message when approving or rejecting workers, much like bonusing works now. It occurred to me that this should be optional though, as some people might not want to send a message for all assignments. Here's the question though: should adding a message be the default, with a flag like "--nomessage" if you don't want to add one, or should there be a flag like "--addmessage" if you DO want to add one? Anyone have strong feelings on this?

**Comment by tobiasgerstenberg** [comment]
thanks for the change! personally, i would always like to tell participants on what grounds i decided to reject their HIT. so for me, it would make sense to have "adding a message" be the default. but i wouldn't mind if the default was the other way around.

**Comment by ebigelow** [comment]
Great, thanks Alex! Like Tobias, I'd think giving a reason would make sense to be the default (>50% of the people I rejected without giving a reason contacted me), though either way would be fine. Adding a flag isn't much trouble -- especially if there is a shortened flag available like "-m".
